build.js

```diff
@@ -69,6 +69,11 @@ var filesToContentHash = function (allFiles, extension) {
         return file.endsWith(extension);
     });
     var hash = fluid.transform(fluid.arrayToHash(extFiles), function (troo, filename) {
+        // When the "inclusive-design/wecount.inclusivedesign.ca" repo installs this project as a dependency and
+        // runs this build script in the subshell (see [npm-explore](https://docs.npmjs.com/cli/v7/commands/npm-explore)),
+        // files in node_modules directory are two levels up in the parent project. These file paths need to be adjusted
+        // to point to the parent node_modules directory.
+        filename = fs.existsSync(filename) ? filename : "../../" + filename;
```

Comment:
I think it's a bit unstable to make guesses about the location of files in node_modules directories in this way. It would work better and then need no special explanation if we 
i) rewrote the topublish.txt file to use symbolic includes for files outside the current project such as 

````
%infusion/src/lib/jquery/core/js/jquery.js
%papaparse/papaparse.js
````
etc,
ii) use a simple string templater such as 

````
var stringTemplate = function (template, vars) {
    var replacer = function (all, match) {
        return vars[match] || "";
    };
    return template.replace(/\${([^\}]*)}/g, replacer);
};
````
together with 
iii) https://www.npmjs.com/package/resolve to find where the module is

Reply:
Comment addressed. Ready for another review.

I noticed when using resolver to find the location of a specific node module, it will return the location of the file where module.exports is defined. For example, `resolver.sync("infusion")` returns `{absolute-local-path}/covid-data-monitor/node_modules/infusion/src/module/fluid.js`. In order to locate the specific files, the logic I used is to remove the starting `%` and pass the rest path to `resolver.sync()`. Please let me know what you think. Thanks.